server/schedulers/adjacent_region.go

```diff
@@ -339,15 +340,7 @@ func (l *balanceAdjacentRegionScheduler) dispersePeer(cluster opt.Cluster, regio
 	if target == nil {
 		return nil
 	}
-	newPeer, err := cluster.AllocPeer(target.GetID())
-	if err != nil {
-		return nil
-	}
-	if newPeer == nil {
-		schedulerCounter.WithLabelValues(l.GetName(), "no-peer").Inc()
```

Comment:
do we need to add this metrics on other?

Reply:
It will record as `create-operator-fail` if alloc ID fail later.